src/apps/content-editor/src/app/views/ItemEdit/components/ItemEditHeader/ConfirmPublishModal.tsx

```diff
@@ -58,7 +63,7 @@ export const ConfirmPublishModal = ({
           Cancel
         </Button>
         <Button
-          autoFocus
```

Comment:
When I tried to revert the initial code with the use of autoFocus only.
Problem: Upon further investigation the problem is not on the autoFocus props, but on the ability to focus `focusRipple`, because if you try to click `save and publish` and press enter it will hit the publish button.

And what I noticed that the focus ripple only works after keyboard interactions like when the user is focus in the textfield and hitting the shortcut key CTRL+P

For reference: (still on latest mui versions this still is not properly working)
https://github.com/mui/material-ui/issues/8438

This is my current fix on the issue above: https://github.com/mui/material-ui/issues/8438#issuecomment-1290291038